modules/openy_features/openy_location/modules/openy_loc_filter/openy_loc_filter.info.yml

```diff
@@ -0,0 +1,11 @@
+name: Open Y Location filter
+description: Provides the ability to limit the list of branches displayed in various popups and filters.
```

Comment:
Better to say `Provides the ability to limit the list of branches displayed in various popups and filters across programs and schedules pages.`